src/dumpType.php

```diff
@@ -3,25 +3,27 @@
 namespace PHPStan;
 
 /**
+ * @template T
  * @phpstan-pure
- * @param mixed $value
- * @return mixed
+ * @param T $value
+ * @return T
  *
  * @throws void
  */
 function dumpType($value) // phpcs:ignore Squiz.Functions.GlobalFunction.Found
```

Comment:
IMO the return type should be `void`.

Reply:
The PR target is to return the value not to make it null or void.